docs/getting_started/quickstart.mdx

```diff
@@ -1,3 +1,95 @@
 ---
 title: 'Quickstart'
----
\ No newline at end of file
+---
+
+## Installation
+
+xorq can be installed using pip:
+
+```bash
+pip install xorq
+```
+
+Or using nix to drop into an IPython shell:
+
+```bash
+nix run github:xorq-labs/xorq
+```
+
+## Quick Start
+
+### Write a pandas UDF
+
+```python
+import xorq as xo
+import xorq.vendor.ibis.expr.datatypes as dt
+
+@xo.udf.make_pandas_udf(
+    schema=xo.schema({"title": str, "url": str}),
+    return_type=dt.bool,
+    name="url_in_title",
+)
+def url_in_title(df):
+    return df.apply(
+        lambda s: (s.url or "") in (s.title or ""),
+        axis=1,
+    )
+
+# Connect to xorq's embedded engine
+con = xo.connect()
+
+# Reference the parquet file
+name = "hn-data-small.parquet"
+expr = xo.deferred_read_parquet(
+    con,
+    xo.options.pins.get_path(name),
+    name,
+).mutate(**{"url_in_title": url_in_title.on_expr})
+
+# Display results
+print(expr.execute().head())
```

Comment:
This will execute the expression `expr` twice when building and running with the CLI. Perhaps we can have:

```suggestion
if __name__ == "__main__":
    print(expr.execute().head())
```

Reply:
that makes sense, but I rather not make the example verbose. I don't think it comes in the way of the example point even if it is run twice.